### ISSUE TITLE
Update did not work for AstroNvim although installed from git + Powershell

### ISSUE THREAD

**Issue by duytrung** [AstroNvim/AstroNvim#1105]
### Question

Hello everyone,

I'm new to AstroNvim and just have a question

I have install AstroNvim through command console using git clone and not quite sure if I have done something wrong but having this message when opening up the nvim:
![image](https://user-images.githubusercontent.com/2046672/195845961-d4b2b600-cecc-4a61-b5cd-baff5ef4001a.png)

These are the command I used from the website "Getting Started"

```
PS C:\Users\...> git clone https://github.com/AstroNvim/AstroNvim $env:LOCALAPPDATA\nvim
Cloning into 'C:\Users\...\AppData\Local\nvim'...
remote: Enumerating objects: 5022, done.
remote: Counting objects: 100% (569/569), done.
remote: Compressing objects: 100% (380/380), done.
remote: Total 5022 (delta 319), reused 313 (delta 165), pack-reused 4453
Receiving objects: 100% (5022/5022), 1.58 MiB | 2.21 MiB/s, done.
Resolving deltas: 100% (3191/3191), done.
PS C:\Users\...> nvim +PackerSync

```
Also using the "intellisense" with not work quite well with the narrowing the words such as when typing "Ae.." it should show a list of functions call with "Ae...." instead of jumping to the first item in the list "AerialClose".

I used the default config, not a custom config as I'm getting to used to with the AstroNvim at the moment.

Thanks for your reply :-)

**Comment by mehalter** [comment]
@duytrung what do you  get if you  go  to the folder you cloned the repo to in your shell and run `git rev-parse --is-inside-work-tree`?

**Comment by mehalter** [comment]
Also @duytrung what version of AstroNvim are you on? We just did some work fixing some bugs in the updater on Windows and that might be the cause of this. Could you try pulling the latest version manually with `git pull` and see if that resolves the issues?

**Comment by alex289** [comment]
Hey, maybe this will help 😅. It may be a problem with the PATH or the installation method.

https://stackoverflow.com/questions/4492979/git-is-not-recognized-as-an-internal-or-external-command)](https://stackoverflow.com/questions/4492979/git-is-not-recognized-as-an-internal-or-external-command

**Comment by mehalter** [comment]
Oh interesting @alex289  I should add a check for this in the updater as well

**Comment by mehalter** [comment]
Oh interesting @alex289  I should add a check for this in the updater as well

**Comment by alex289** [comment]
Would definitely not hurt

**Comment by duytrung** [comment]
@mehalter I have the following item in the folder %appdata%\Local\nvim

```

 Directory of C:\Users\...\AppData\Local\nvim

14/10/2022  11:07 PM    <DIR>          .
14/10/2022  11:07 PM    <DIR>          ..
14/10/2022  05:14 AM    <DIR>          .github
14/10/2022  05:14 AM                90 .gitignore
14/10/2022  05:14 AM               374 .luacheckrc
14/10/2022  05:14 AM               177 .stylua.toml
14/10/2022  05:14 AM    <DIR>          colors
14/10/2022  05:14 AM               208 config.ld
14/10/2022  05:14 AM               705 init.lua
14/10/2022  05:14 AM            35,149 LICENSE
14/10/2022  05:14 AM    <DIR>          lua
14/10/2022  05:14 AM             1,862 packer_snapshot
               7 File(s)         38,565 bytes
               5 Dir(s)   8,526,725,120 bytes free
```
Here is the contents of git rev-parse as your instruction:
```
C:\Use...\AppData\Local\nvim>git rev-parse --is-inside-work-tree
fatal: not a git repository (or any of the parent directories): .git

```

Also,
1. Using "AstroVersion" doesn't work in this case. It didn't show up any version as in my Linux Ubuntu distro.
2. Using "git pull" showing similar result as "git rev-parse".
My current system is Windows 10.

Thanks alot :-)

**Comment by mehalter** [comment]
Yeah, I'm not sure exactly what happened with your set up, but the AstroNvim folder is not a git repository. So `git` is unable to do anything with it. Are you sure you rand `git clone`? Maybe you just downloaded the source code from GitHub or something?

**Comment by alex289** [comment]
The .git folder is not present so it's either deleted or something, or manually downloaded

**Comment by mehalter** [comment]
I would recommend deleting and recloning the repo in the command line and make sure that when you run the command I sent it doesn't say that it's not a git repository. You can also verify this by checking to see if there is a `.git` folder as described by @alex289

**Comment by mehalter** [comment]
Since we have identified the root cause, I'm going to go ahead and close this. Hopefully we have provided enough guidance to get the `git` repo set up enough @duytrung . Let me know if you still have any other questions and we can keep discussing on this issue even with it closed.

Also I have gone ahead and added a check at the beginning of the update path to check if the `git` command is available and have pushed that.
